src/DssSpell.sol

```diff
@@ -45,51 +49,100 @@ contract DssSpellAction is DssAction, DssSpellCollateralOnboardingAction {
-    address constant DOO_WALLET             = 0x3B91eBDfBC4B78d778f62632a4004804AC5d2DB0;
-    address constant FLIPSIDE_CRYPTO_WALLET = 0x62a43123FE71f9764f26554b3F5017627996816a;
+    address constant MCD_END   = 0x0e2e8F1D1326A4B9633D96222Ce399c708B19c28;
+    address constant MCD_CURE  = 0x0085c9feAb2335447E1F4DC9bf3593a8e28bdfc7;
+    address constant MCD_FLASH = 0x60744434d6339a6B27d73d9Eda62b6F66a0a04FA;
+
+    address immutable MCD_VAT          = DssExecLib.vat();
+    address immutable MCD_VOW          = DssExecLib.vow();
+    address immutable MCD_CAT          = DssExecLib.cat();
+    address immutable MCD_DOG          = DssExecLib.dog();
+    address immutable MCD_SPOT         = DssExecLib.spotter();
+    address immutable MCD_POT          = DssExecLib.pot();
+    address immutable MCD_ESM          = DssExecLib.esm();
+    address immutable ILK_REGISTRY     = DssExecLib.reg();
+    address immutable MCD_END_OLD      = DssExecLib.end();
+    address immutable MCD_FLASH_LEGACY = DssExecLib.getChangelogAddress("MCD_FLASH");
+    address immutable MCD_GOV          = DssExecLib.mkr();
+
+    address immutable SES_WALLET = 0x87AcDD9208f73bFc9207e1f6F0fDE906bcA95cc6;
 
     function actions() public override {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralOnboardingAction
-        onboardNewCollaterals();
-
-        // MOMC Proposal
-        // https://vote.makerdao.com/polling/QmTmehbz#poll-detail
-
-        // Lower the WBTC-A Stability Fee from 3.25% to 2.25%.
-        DssExecLib.setIlkStabilityFee("WBTC-A", TWO_TWO_FIVE_PCT_RATE, true);
-
-        // Lower the WBTC-B Stability Fee from 4.5% to 4.0%.
-        DssExecLib.setIlkStabilityFee("WBTC-B", FOUR_PCT_RATE, true);
-
-        // Recognized Delegate Payments
-        // https://forum.makerdao.com/t/recognized-delegate-compensation-breakdown-april-2022/14935
-
-        DssExecLib.sendPaymentFromSurplusBuffer(FLIP_FLOP_FLAP_WALLET,  12_000);
-        DssExecLib.sendPaymentFromSurplusBuffer(ULTRASCHUPPI_WALLET,    12_000);
-        DssExecLib.sendPaymentFromSurplusBuffer(FEEDBLACK_LOOPS_WALLET, 12_000);
-        DssExecLib.sendPaymentFromSurplusBuffer(MAKERMAN_WALLET,        10_929);
-        DssExecLib.sendPaymentFromSurplusBuffer(ACRE_INVEST_WALLET,      9_347);
-        DssExecLib.sendPaymentFromSurplusBuffer(MONETSUPPLY_WALLET,      8_626);
-        DssExecLib.sendPaymentFromSurplusBuffer(JUSTIN_CASE_WALLET,      7_522);
-        DssExecLib.sendPaymentFromSurplusBuffer(GFX_LABS_WALLET,         6_607);
-        DssExecLib.sendPaymentFromSurplusBuffer(DOO_WALLET,                351);
-        DssExecLib.sendPaymentFromSurplusBuffer(FLIPSIDE_CRYPTO_WALLET,    265);
-
-        DssExecLib.setChangelogVersion("1.12.1");
+        // onboardNewCollaterals();
+
+        // -------------------------- Cure + New End ---------------------------
```

Comment:
can we add links to the relevant polls or forum posts? like: https://github.com/makerdao/spells-mainnet/blob/d7c970e042a1af3b6f5b0c478411425615321eeb/archive/2022-05-11-DssSpell/DssSpell.sol#L69-L70

Reply:
I mentioned this to Tal the other day, in general I do not add the specific links to each action. Details are well explained in the summary linked above. But if you feel strong about adding it, we can look for the links and put in the code.

Reply:
I have just been used to seeing them and they were especially helpful when we were authoring the spell before the main contents were available, but maybe less important now.  I flagged it because I thought it was convention... if not, you can disregard